JavaScript/JavaScript.sublime-syntax

```diff
@@ -1834,9 +1834,9 @@ contexts:
       captures:
         1: punctuation.definition.variable.js
         2: variable.other.readwrite.js
-
+      pop: true
     - match: (?=\[)
-      push: computed-property-name
+      set: computed-property-name
```

Comment:
Would it make sense to directly include the context?

```yaml
    - include: computed-property-name
```

_Note: It would apply to the previous `method-name` context as well._

Reply:
Good call. I've also done this in `object-property-name`.